File: Python Code/Daily_Flash_Code_0/Week9/21Mar/53-DailyFlash_Solutions/21_March_Solution_Three/C/prog3.c
#include<stdio.h>

int main() {
	int num, n = 0;
	scanf("%d",  &num);
	int temp = num;
	while(temp != 0) {
		n++;
		temp /= 10;
	}
	int arr[n];
	int i = 0;
	while(num != 0) {
		arr[i] = num % 10;
		num /= 10;
		i++;
	}
	float sum = 0;
	for(int i = 0; i < n; i++) {
		sum += arr[i];	
	}
	printf("sum : %f\navg : %f", sum, sum / n);

}

File: Python Code/Daily_Flash_Code_0/Week9/21Mar/53-DailyFlash_Solutions/21_March_Solution_Three/Java/prog3.java
import java.util.Scanner;
class Demo {
	public static void main(String[] args) {
		int num;
		Scanner sc = new Scanner(System.in);
		num = sc.nextInt();
		int temp = num, n = 0;
		while(temp != 0) {
		
			temp /= 10;
			n++;
		}
		int [] arr = new int[n];
		int i = 0;
		while(num != 0) {
			arr[i] = num % 10;
			num /= 10;
			i++;
		}
		int sum = 0;
		for(int j = 0; j < n; j++) {
			sum += arr[j];
		}
		System.out.print("sum : " + sum + "\navg : " + (float)sum / n);

	}
}
